packages/stark-ui/src/modules/app-logout/components/app-logout.component.ts

```diff
@@ -0,0 +1,64 @@
+import { Component, Inject, Input, OnInit, ViewEncapsulation } from "@angular/core";
+
+import {
+	STARK_LOGGING_SERVICE,
+	STARK_ROUTING_SERVICE,
+	STARK_APP_CONFIG,
+	StarkLoggingService,
+	StarkRoutingService,
+	StarkApplicationConfig,
+	STARK_SESSION_SERVICE,
+	StarkSessionService
+} from "@nationalbankbelgium/stark-core";
+
+/**
+ * Name of the component
+ */
+const componentName: string = "stark-app-logout";
+
+/**
+ * Component to display the application's logout button
+ */
+@Component({
+	selector: "stark-app-logout",
+	templateUrl: "./app-logout.component.html",
+	encapsulation: ViewEncapsulation.None
+})
+export class StarkAppLogoutComponent implements OnInit {
+	/**
+	 * Desired icon of the logout button
+	 */
+	@Input()
+	public icon: string = "power";
```

Comment:
Can you add the "power" literal type so it is clearer for the user? Basically something like this:

`public icon: "power" | string = "power";`